multicluster/Makefile

```diff
@@ -2,7 +2,7 @@
 # Image URL to use all building/pushing image targets
 IMG ?= antrea/antrea-mc-controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+CRD_OPTIONS ?= "crd:trivialVersions=true,allowDangerousTypes=true,preserveUnknownFields=false"
```

Comment:
/multicluster codebase uses controller-gen instead of kustomize for generating the CRD yamls. For controller-gen, float value is not allowed by default as it is considered dangerous https://github.com/kubernetes-sigs/controller-tools/issues/245. 
```
 floats are highly discouraged (and disallowed in official kubernetes APIs) because they won't round-trip properly, can't actually represent certain numbers exactly, etc (all the normal reasons floats are usually not what you want in a program). We recommend using resource.Quantity or int32 for your numeric types, in accordance with the Kubernetes API conventions.
```
However for this PR, the ResourceExport/Import refers to ACNP type definition and the priority field in ACNP is of type float64. Hence without this flag CRD yamls for ResourceExport/Import wouldn't generate properly. I will add a comment and open an issue to track this. Currently I'm thinking to update acnp.spec.priority from `float` to `reaource.Quantity` in the next API spec bump